src/diffusers/pipelines/blip_diffusion/pipeline_blip_diffusion.py

```diff
@@ -0,0 +1,383 @@
+# Copyright 2023 Blip Diffusion authors and The HuggingFace Team. All rights reserved.
```

Comment:
Let's stay consistent. In one place we write "Salesforce" and here we are using "Blip Diffusion". Let's maybe go with Salesforce.